bin/seqchksum_merge.pl

```diff
@@ -159,7 +159,9 @@ sub initialise_outrows {
 	}
 
 	# convert non-comment rows into arrays
-	$init_rows = [ map { $_ !~ /^\#/smx? [ (split /\t/smx, $_) ]: $_; } @{$inrows} ];
+        ##no critic (RegularExpressions::ProhibitUselessTopic)
+	$init_rows = [ map { ($_ !~ /^\#/smx) ? [ (split /\t/smx ) ]: $_; } @{$inrows} ];
```

Comment:
How about ```$init_rows = [ map { /^\#/smx ? $_ : [ split /\t/smx ] } @{$inrows} ];``` to avoid the ```no critic```. Dropped a set of parentheses as well.